metron-platform/metron-common/src/main/java/org/apache/metron/common/utils/ConversionUtils.java

```diff
@@ -35,6 +35,14 @@ protected ConvertUtilsBean initialValue() {
     }
   };
 
+  public static <T> T convertOrFail(Object o, Class<T> clazz) {
+    if (clazz.isInstance(o)) {
```

Comment:
Good point. I'm removing this entirely. I'll just use casting in place, e.g.

**Example 1**
`String val = (String) map.get("foo"); // throws class cast exception on failure, which is what we want`

**Example 2**
```
Map<Object, Object> a = new HashMap() {{
    put("hello", "world");
    put(1, 2);
}};

Map<String, Object> b = new HashMap() {{
    put("a", a);
}};
Map<Object, Object> c = (Map) b.get("a"); // throws class cast exception if not a Map
Map<String, String> d = new HashMap<>();
for (Map.Entry<Object, Object> entry : c.entrySet()) {
    d.put((String) entry.getKey(), (String) entry.getValue()); // throws class cast exception. also what we want
}
```